content/sensu-go/5.12/api/namespaces.md

```diff
@@ -14,6 +14,8 @@ menu:
 - [The `/namespaces/:namespace` API endpoint](#the-namespacesnamespace-api-endpoint)
   - [`/namespaces/:namespace` (PUT)](#namespacesnamespace-put)
   - [`/namespaces/:namespace` (DELETE)](#namespacesnamespace-delete)
+- [The `/user-namespaces` API endpoint](#the-user-namespaces-api-endpoint)
+  - [`/user-namespaces` (GET)](#user-namespaces-put)
```

Comment:
This appears to have the wrong verb mapping (GET) -> #user-namespaces-put

Reply:
Fixed! Nice catch @echlebek.